canonical-web-and-design/ubuntu.com
# Remove 5.4 kernel

## Done

- Removes 5.4 from kernel release cycle chart.

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/about/release-cycle
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare against [copydoc](https://docs.google.com/document/d/1-l3B7AoyDoETaL80UhlfEWCVKmMWPdy3zoF5LpsGcYQ/edit)

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/4377
